core_gl/src/utility/RenderUtils.cpp

```diff
@@ -102,6 +96,41 @@ bool RenderUtils::CreateShader(std::shared_ptr<glowl::GLSLProgram>& out_shader_p
 }
 
 
+bool RenderUtils::CreateShader(
+    glowl::GLSLProgram** out_shader_ptr, const std::string& vertex_code, const std::string& fragment_code) {
```

Comment:
Would a `glowl::GLSLProgram&` work here too?

Reply:
Obsolete due to commit 0e88026